wcbench.sh

```diff
@@ -480,11 +545,11 @@ uninstall_odl()
 {
     if [ -d $ODL_DIR ]; then
         echo "Removing $ODL_DIR"
-        rm -rf $ODL_DIR
+        sudo rm -rf $ODL_DIR
```

Comment:
So, ideally we'd not run ODL as root. For example, in the RPM and Puppet mod we create an `odl` user to own the ODL dir and the ODL process. Here we're just defaulting to the `vagrant` user, which should be fine. But yeah, running as root should be avoided for security reasons.